gzipi/lib.py

```diff
@@ -146,41 +169,139 @@
 _BYTES_IN_KiB = 1024
 
 
-def _is_valid_gzip_header(gzip_header):
+class Compression(enum.Enum):
+    NONE = None
+    GZIP = 'gzip'
+    ZSTD = 'zstd'
+
+
+SUPPORTED_COMPRESSIONS = {Compression.GZIP.value, Compression.ZSTD.value, }
+
+
+def get_compression_format(path: str) -> Optional[str]:
```

Comment:
There are many "get" functions here, it's a little bit confusing.

```suggestion
def assume_compression_from_filename(path: str) -> Optional[str]:
```